index.js

```diff
@@ -2,8 +2,8 @@ const path = require('path')
 const fs = require('fs')
 const qs = require('querystring')
 
-// create global paramPattern but define it in the router to allow custom char for parameterized routes
-let paramPattern
+
+const paramPattern = /:([^\/%]+)/
```

Comment:
I don't understand this regex.

We want :params or %params.
So regex become /(:|%)([^\/]+)/ ?